packages/vuetify/src/components/VInput/VInput.tsx

```diff
@@ -112,7 +112,7 @@ export const VInput = genericComponent<new <T>() => {
       )
       const hasDetails = !props.hideDetails || (
         props.hideDetails === 'auto' &&
-        hasMessages
+        (hasMessages || !!slots.details)
```

Comment:
should this also include `slots.messages`? While `slots.details` is rendering, it has no content. Not sure if this is related.